packages/ui/ErrorBoundary.tsx

```diff
@@ -20,7 +20,7 @@ class ErrorBoundary extends React.Component<
       // Error path
       return (
         <div>
-          <h2>Something went wrong.</h2>
+          <h2>{this.props.message || "Something went wrong."}</h2>
```

Comment:
Support custom message in ErrorBoundary

Reply:
Should "Something went wrong." be translated? 🤔

Reply:
I think it should be. Not sure how to re-use a hook(useLocale) code in class-based component